### DISCUSSION TITLE
window state for climate groups

### DISCUSSION THREAD

**Discussion by Peet1985** [SukramJ/aiohomematic#1704] [General]
Hello,

Is it possible to display and change the window state for climate groups? In the iobroker I was able to address the parameter directly in HA, but unfortunately it is not shown to me.

Thanks and greetings
Peet

**Comment by SukramJ** [comment]
Just add `window_state` to [unignore](https://github.com/danielperna84/hahomematic/blob/devel/docs/unignore.md#using-the-ui)

**Comment by Peet1985** [comment]
Cool, Thanks
